shared/src/main/scala/uk/gov/homeoffice/drt/arrivals/ApiArrivalsWithSplits.scala

```diff
@@ -95,10 +95,10 @@ case class ApiFlightWithSplits(apiFlight: Arrival, splits: Set[Splits], lastUpda
     }
   }
 
-  def isWithinThreshold(apiSplits: Splits): Boolean = apiFlight.ActPax.forall { actPax =>
+  def isWithinThreshold(apiSplits: Splits): Boolean = {
     val apiPaxNo = apiSplits.totalExcludingTransferPax
     val threshold: Double = 0.05
-    val portDirectPax: Double = actPax - apiFlight.TranPax.getOrElse(0)
+    val portDirectPax: Double = apiFlight.bestPcpPaxEstimate.getPcpPax.getOrElse(0).toDouble
```

Comment:
This needs to specifically be the LiveFeedSource pax value. If we update the `bestPcpPaxEstimate` to favour API over Live then this function would always return true